### PR TITLE
Usage example for readme

### PR DESCRIPTION
**Pull request by andrii-balitskyi**
- usage example added to readme
- url added to ConnectWebview type

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,6 +2,43 @@
...
+      access_codes: [
+        {
+          access_code_id: "some-access-code-1",
+          name: "Some Access Code",
+          code: "1234",
+          type: "ongoing",
+          created_at: "2022-02-01T12:57:40.843Z",
+        },
+      ],
+    }
```
Comment (seveibar) [inline README.md:38]: it's more conventional to write

```ts
const accessCodeList = seam.accessCodes.list(someLockId)
console.log(accessCodeList)
/*
{
    access_codes: [
      {
        access_code_id: "some-access-code-1",
        name: "Some Access Code",
        code: "1234",
        type: "ongoing",
        created_at: "2022-02-01T12:57:40.843Z",
      },
    ],
  }
*/
```

File: README.md [hunk 2]
```diff
@@ -2,6 +2,43 @@
 
 Control locks, lights and other internet of things devices with Seam's simple API. Check out [the documentation](./docs/modules.md) or [some examples](./examples).
 
+
+## Usage
+
+```ts
+import Seam from seamapi
+
+// export SEAM_API_KEY=***
```
Comment (seveibar) [inline README.md:11]:
```suggestion
// Seam will automatically use the SEAM_API_KEY environment variable if you
// don't provide an api_key to `new Seam()`
```

**Review by seveibar — CHANGES_REQUESTED** [review]

**Review by seveibar — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -2,6 +2,44 @@
...
+import Seam from seamapi
+
+const seam = new Seam()
+// Seam will automatically use the SEAM_API_KEY environment variable if you
+// don't provide an api_key to `new Seam()`
+
+const someLock = seam.locks.list().devices[0]
+const someLockId = someLock.device_id
+
+seam.locks.lockDoor(someLockId)
```
Comment (codetheweb) [inline README.md:18]: These API calls should be `await`ed.

**Review by codetheweb — COMMENTED** [review]
Looks pretty good to me!

**Review by codetheweb — APPROVED** [review]

**Pull request merged**